be/src/olap/tablet_manager.cpp

```diff
@@ -628,6 +627,22 @@ bool TabletManager::get_tablet_id_and_schema_hash_from_path(const std::string& p
     return false;
 }
 
+bool TabletManager::get_rowset_id_from_path(const std::string& path, RowsetId* rowset_id) {
+    std::string pattern = "/data/\\d+/\\d+/\\d+/(\\d+)_.*";
+    std::regex rgx (pattern.c_str());
```

Comment:
You should make this static to avoid constructing this multiple times.

Reply:
ok